### ISSUE TITLE
Renaming "More Controls"?

### ISSUE THREAD

**Issue by arouinfar** [phetsims/a11y-research#110]
During the a11y sprint, we decided to try out "More Controls" instead of "Control Panel". 

However, when reviewing the scene summary for Coulomb's Law (https://github.com/phetsims/coulombs-law/issues/43), @emily-phet had some further thoughts.

> [General question, scope is broader than this specific sim] What about "Control Area(s)". This would read a little better at the start of the scene summary ("Play Area and Control Area" or "Play Area and Control Areas"). This would also allow a grouping of controls that is more aligned with the visual display - e.g., play/pause buttons are typically grouped together, reset and sound on/off are historically grouped together (sound may move into the nav bar as a global control...but I think you get the idea). This would avoid the awkward situation of a student thinking there is one set of "More Controls" someone one the screen, while another student sees their distributed around. Thoughts?

For more context, here's the scene summary
> This is an interactive sim. It changes as you play with it. It has a Play Area and **More Controls**. The Play Area has two charged spheres, Charge 1 (q1) and Charge 2 (q2), currently separated by {{distance}} centimeters. Each sphere is held by a small robot. Spheres can be moved closer or further from one another, and the charge of each sphere can be increased or decreased. **More Controls** has checkboxes to display forces or change to scientific notation, and a button to reset the sim.

I think "Control Area" (plural doesn't seem necessary) is a better parallel to "Play Area" and has less grammatical baggage than "More Controls", but we should have a group discussion about whether or not to make this change.

Tagging potentially interested parties @terracoda @amanda-phet @ariel-phet @oliver-phet @jessegreenberg @mbarlow12 .

**Comment by amanda-phet** [comment]
I REALLY like Control Area as the name! It makes perfect sense to me. I can see sims that have less going on in the play area really benefitting from this.

**Comment by terracoda** [comment]
I agree with @amanda-phet, I really like "Control Area", too. I can't believe we didn't come up with it before. Way to go @emily-phet for seeing the forest for the trees!

**Comment by arouinfar** [comment]
@emily-phet took a quick poll in the keyboard navigation meeting this morning, and the consensus was to move forward with "Control Area". 

Folks working on descriptions may want to update any relevant a11y docs @terracoda @amanda-phet.

**Comment by zepumph** [comment]
There are still places in the code that say "Control Panel".

**Comment by zepumph** [comment]
Tagging @jessegreenberg, closing
